src/core/Akka.FSharp/FaultHandling.fs

```diff
@@ -0,0 +1,135 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="FaultHandling.fs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Akka.FSharp
+
+open Akka.Actor
+open System
+open Microsoft.FSharp.Quotations
+open Microsoft.FSharp.Linq.QuotationEvaluation
+
+
+[<AutoOpen>]
+module Logging = 
```

Comment:
Does this really go with the supervisor strategies below? If not, kick it out into its own file?